user-group/humhub-mass-import-inline.html

```diff
@@ -0,0 +1,164 @@
+            .ExternalClass div {
+                line-height: 100%;
+            }
+            .apple-link a {
+                color: inherit !important;
+                font-family: inherit !important;
+                font-size: inherit !important;
+                font-weight: inherit !important;
+                line-height: inherit !important;
+            }
+            .btn-primary table td:hover {
+                background-color: #44474F !important;
+            }
+            .btn-primary a:hover {
+                background-color: #44474F !important;
+                border-color: #44474F !important;
+            }
+        }
+    </style>
+</head>
+<body class="" style="background-color:#EAECEC;font-family:sans-serif;-webkit-font-smoothing:antialiased;font-size:14px;line-height:1.6;margin:0;padding:0;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;">
+    <table border="0" cellpadding="0" cellspacing="0" class="body" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;background-color:#EAECEC;width:100%;">
+        <tr>
+            <td style="font-family:sans-serif;font-size:14px;vertical-align:top;">&nbsp;</td>
+            <td class="container" style="font-family:sans-serif;font-size:14px;vertical-align:top;display:block;max-width:580px;padding:10px;width:580px;Margin:0 auto !important;">
+                <div class="content" style="box-sizing:border-box;display:block;Margin:0 auto;max-width:580px;padding:10px;">
+                    <!-- START CENTERED WHITE CONTAINER -->
+                    <span class="preheader" style="color:transparent;display:none;height:0;max-height:0;max-width:0;opacity:0;overflow:hidden;mso-hide:all;visibility:hidden;width:0;">Your Blackboard Ally User Group account is ready.</span>
+                    <table class="main" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;background-color:#FFFFFF;border-radius:3px;width:100%;">
+                        <!-- START MAIN CONTENT AREA -->
+                        <tr>
+                            <td class="wrapper" style="font-family:sans-serif;font-size:14px;vertical-align:top;box-sizing:border-box;text-align:center;">
+                                <table border="0" cellpadding="0" cellspacing="0" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;width:100%;">
+                                    <tr>
+                                        <td class="header" style="font-family:sans-serif;font-size:14px;vertical-align:top;background-color:#191E25;box-sizing:border-box;padding:20px;">
+                                            <h1 style="color:#191F25;font-family:sans-serif;font-weight:normal;line-height:1.4;margin:0;text-align:center;">
+                                                <img alt="Ally logo" height="70px" src="http://ally.ac/assets/img/ally-logo.png" title="Ally" style="border:none;-ms-interpolation-mode:bicubic;max-width:100%;"/></h1>
```

Comment:
Minor: Having the `</h1>` on this line has caused the indentation to be a bit askew.